src/pages/preferences/preferencesTabs/cliTab/AuthenticationCard.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Button, { Variant } from "@leafygreen-ui/button";
+import { Subtitle } from "@leafygreen-ui/typography";
+import Styled from "@emotion/styled";
```

Comment:
is it always uppercased in your files? it's always lowercased in arjun and i's

Reply:
Nice Catch I did not notice it I did a Find and Replace it looks like for some reason i only used upper case styled in this pr.